### PR TITLE
vuln(NSWG-ECO-415): react-svg

### PR DESCRIPTION
**Pull request by ronperris**

### CODE REVIEW

File: vuln/npm/415.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "updated_at": "2018-04-27",
+  "publish_date": "2018-04-27",
+  "author": "Ron Perris",
+  "module_name": "react-svg",
+  "cves": [],
+  "vulnerable_versions": "<=2.2.17",
+  "patched_versions": ">=2.2.18",
+  "recommendation": "Update react-svg module to 2.2.18 or higher.",
+  "references": "https://github.com/tanem/react-svg/pull/57",
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H",
```
Comment (lirantal) [inline vuln/npm/415.json]: is it really a score of 10?
I don't think it causes availability issues. can you update accordingly?

Comment (ronperris) [reply vuln/npm/415.json]: Maybe I'm wrong but I'm thinking about the confidentiality, integrity and availability of information that is on the page affected by the XSS vulnerability.

This arbitrary code execution vulnerability does affect the availability of information. If an attack can execute arbitrary code on the web page they can deny access to the information on that page.

Let me know where I'm wrong in my thing here. @lirantal

Comment (lirantal) [reply vuln/npm/415.json]: I understand availability disruption in terms of complete access to the service, hence it is something that would probably be affected on the server side. The fact you will deny access to the information on the page already manifests in the Integrity portion of the score.

**Comment by ronperris** [comment]
@lirantal Thanks for the help with the CVSS score! Is this better?

**Comment by ronperris** [comment]
@lirantal Are you waiting for me to make a change before merging this one?

**Comment by lirantal** [comment]
No, it's ok, was AFK for a while.
Thanks Ron!

**Pull request merged**